lib/__tests__/postal-address.test.ts

```diff
@@ -342,4 +342,15 @@ describe('Propagation', () => {
     expect(myAddress.toObject().lastName).toBe('Doe')
     expect(myAddress.toObject().secondLastName).toBe('Smith')
   })
+
+  it('Propagation of changes to related properties overwrites with latest invoked even if both present', () => {
```

Comment:
I added this to make the behavior clear. The latest invoked item will be the one that overrides the explicit calling of the propagated, related item.